wasm/lib/src/vm_class.rs

```diff
@@ -24,7 +24,7 @@ pub(crate) struct StoredVirtualMachine {
     pub scope: RefCell<Scope>,
     /// you can put a Rc in here, keep it as a Weak, and it'll be held only for
     /// as long as the StoredVM is alive
-    held_rcs: RefCell<Vec<Rc<dyn HeldRcInner>>>,
+    held_objects: RefCell<Vec<PyObjectRef>>,
```

Comment:
Yeah, this is a better system than what I had. You can just remove `HeldRcInner` now.